nibabel/freesurfer/mghformat.py

```diff
@@ -84,6 +92,7 @@ class MGHHeader(object):
 
     def __init__(self,
                  binaryblock=None,
+                 endianness='>',
```

Comment:
I did this because of places where `LabeledWrapStruct` calls `__init__` assuming the type signature of the base class. I can instead make sure to override any methods that make such calls, just figured this would be a little shorter.

Reply:
I played with the idea of making WrapStruct have a parent or similar that had forced endianness, but I seem to remember it got a bit messy.